src/Schema/Keywords/Type.php

```diff
@@ -52,19 +49,17 @@ public function validate($data, string $type, ?string $format = null) : void
                 }
                 break;
             case CebeType::BOOLEAN:
-                $stringifiedBool = is_scalar($data) && preg_match('#^(true|false)$#i', (string) $data);
-                if (! is_bool($data) && ! $stringifiedBool) {
+                if (is_string($data) || ! is_bool($data)) {
```

Comment:
Looks redundant. If data is string it should be !is_bool as well, so right part is always true when left part is true and we can keep just right part

Reply:
Fixed